src/ScottPlot4/ScottPlot/Plottable/ScatterPlot.cs

```diff
@@ -426,6 +427,27 @@ public void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false)
                 if ((MarkerSize > 0) && (MarkerShape != MarkerShape.none))
                 {
                     MarkerTools.DrawMarkers(gfx, points, MarkerShape, MarkerSize, MarkerColor, MarkerLineWidth);
+                    if (DataPointLabels != null)
```

Comment:
Let's use { } to wrap the full contents of this if statement

Reply:
Also, this if statement should be _outside_ the if statement two lines up about markers. I think it makes sense to always show the labels if they are populated (not only if marker size is >0)